doc/manual/source/protocols/tarball-fetcher.md

```
@@ -1,33 +1,27 @@
-server must return an [HTTP `Link` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link) with the `rel` attribute set to
-`immutable`, as follows:
+# Lockable HTTP Protocol
+
+Nix can fetch a flake input over HTTP, or from the file system for `file://` URLs.
+If the server does not support the Lockable HTTP Protocol, the user must make sure that the URL always gives the same contents.
+
+An HTTP server can return an "immutable" URL for the lock file.
+Then the user can put an input in `flake.nix` that asks for the most recent version of a resource (for example, `https://example.org/hello/latest.tar.gz`), while `flake.lock` records a URL whose contents do not change (for example, `https://example.org/hello/<revision>.tar.gz`).
+To do this, the server must send an [HTTP `Link` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link) with the `rel` attribute set to `immutable`, as follows:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)